### DISCUSSION TITLE
en/chapter_preface/summary/

### DISCUSSION THREAD

**Discussion by giscus** [krahets/hello-algo#1047] [Announcements]
# en/chapter_preface/summary/

Data Structures and Algorithms Crash Course with Animated Illustrations and Off-the-Shelf Code

https://www.hello-algo.com/en/chapter_preface/summary/

**Comment by leone2016** [comment]
Thanks for this resources.

**Comment by maxlesscode** [comment]
Thanks for awesome work !!
